src/components/Navbar.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div id="navbar">
+        <el-button type="danger" class="button">
+            <router-link to="/#">Overview</router-link>
+        </el-button>
+
+        <el-button type="danger" class="button">
+            <router-link to="/#">Map</router-link>
+        </el-button>
+
+        {{points}} Points
+
+        <el-button type="danger" class="button">
+            <router-link to="/#">Reports</router-link>
+        </el-button>
+
+        <el-button type="danger" class="button">
+            <router-link to="/#">Tribe</router-link>
+        </el-button>
+    </div>
+</template>
+
+<script>
+    export default {
+        name: "Navbar",
+        computed: {
+            points() {
+                return this.$store.getters.points
+            }
+        }
```

Comment:
it will works, but besides that the convention is that u should use `mapGetters` method which is provided by vuex package.
So, at the begining you should `import mapGetters from 'vuex'`, then use destructurized mapGetters (`...mapGetters(['exampleGetter'])`) on computed block to unlock access to getters which only you need.

docs: https://vuex.vuejs.org/guide/getters.html#the-mapgetters-helper